into/backends/json.py

```diff
@@ -192,4 +193,18 @@ def json_dumps(dt):
     return dt.isoformat()
 
 
+@convert.register(chunks(list), chunks(JSON))
+def convert_glob_of_jsons_into_chunks_of_lists(jsons, **kwargs):
+    def _():
+        return concat(convert(chunks(list), js, **kwargs) for js in jsons)
+    return chunks(list)(_)
+
+
+@convert.register(chunks(Iterator), chunks(JSONLines))
+def convert_glob_of_jsons_into_chunks_of_lists(jsons, **kwargs):
```

Comment:
Hrm, interesting thought.  No strong reason why it isn't there other than I found that it wasn't necessary (at the time.)